sdk/src/main/java/com/google/cloud/dataflow/sdk/runners/DataflowPipelineRunner.java

```diff
@@ -424,6 +424,12 @@ public static DataflowPipelineRunner fromOptions(PipelineOptions options) {
     return super.apply(new AssignWindows<>(transform), input);
   }
 
+  private String debuggerMessage(String projectId, String uniqifier) {
+    return String.format("To view the debugger console, go to "
```

Comment:
Branding issue here. For example: "To debug your job, visit Google Cloud Debugger at: ".

From their website: "Visit the Debug page of the Cloud Platform Console to use Google Cloud Debugger."